File: Python Code/Daily_Flash_Code_0/Week9/21Mar/53-DailyFlash_Solutions/21_March_Solution_Three/Python/prog3.py
import array
num = int(input())
arr = array.array("i", [])
while(num != 0):
    arr.append(num % 10)
    num = int(num / 10)
s = 0
for i in arr:
    s += i
print("sum =", s)
print("avg =", s/len(arr))

File: Python Code/Daily_Flash_Code_0/Week9/20Mar/52-DailyFlash_Solutions/20-Mar/C++/prog3.cpp
#include<iostream>

int main() {
	int num;
	std::cin >>num;
	int temp = num, n = 0;
	while(temp != 0) {
		n++;
		temp /= 10;
	}
	int arr[n];
	int i = 0;
	while(num != 0) {
		arr[i] = num % 10;
		num /= 10;
		i++;
	}
	for(int i = 0; i < n; i++) {
		std::cout << arr[i];
	}

}
